### PR TITLE
create release checklist

### PR DESCRIPTION
**Pull request by Loquacity**

### CODE REVIEW

File: .github/release_issue_template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Documentation Release Checklist
+
+Use this issue template for documentation releases.
+
+## Packaging Day:
+
+- [ ] Ensure all relevant PRs are merged (and backported where required)
```
Comment (keichwa) [inline .github/release_issue_template.md:7]: How?  Is it enough to check https://github.com/uyuni-project/uyuni-docs/pulls?  Do we also have to check the board?

File: .github/release_issue_template.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- [ ] Cut a new branch from master using syntax `manager-x.y-betaz`
+- [ ] Build both SUMA and Uyuni docs from release branch and visually check output
```
Comment (keichwa) [inline .github/release_issue_template.md:9]:
```suggestion
- [ ] Build both SUMA and Uyuni docs from release branch and visually check output: https://github.com/uyuni-project/uyuni-docs/wiki/building-documentation
```
Or do we have a better recipe?

Comment (Loquacity) [reply .github/release_issue_template.md:9]: i was thinking more `make antora-suma` and `make anotra-uyuni` ;)

**Comment by keichwa** [comment]
On 2020-06-12 11:11, Lana Brindley wrote:

> i was thinking more `make antora-suma` and `make anotra-uyuni` ;)

Good that I asked ;)

-- 
Karl Eichwalder                    SUSE Software Solutions Germany GmbH
                                    Maxfeldstraße 5
                                    90409 Nürnberg, Germany
Geschäftsführer: Felix Imendörffer (HRB 36809, AG Nürnberg)

**Pull request merged**